protos/google/spanner/v1/type.proto

```diff
@@ -74,6 +74,17 @@ enum TypeCode {
   // Encoded as `list`, where list element `i` is represented according
   // to [struct_type.fields[i]][google.spanner.v1.StructType.fields].
   STRUCT = 9;
+
+  // Encoded as `string`, in decimal format or scientific notation format.
+  // <br>Decimal format:
+  // <br>`[+-]Digits[.[Digits]]` or
```

Comment:
One minor problem with using the `big.js` library for this, is that it does not seem to support exactly the same formats as Cloud Spanner. Most notably, Cloud Spanner allows a leading + sign for positive numbers, while `big.js` seems not to accept that. As long as Cloud Spanner will return positive values without a leading + sign, it should not really be a big problem, but it will mean that a user cannot in all cases use the same notation in the Node client as what could have been used directly with Cloud Spanner.

Reply:
@larkee FYI.

Ah ok, thanks @olavloite for pointing this out. I did not think about this. AFAIK we chose this to be consistent with BigQuery since the backend representation is the same in both Spanner and BigQuery. I don't think Spanner will actually return values with a leading + sign but I will check with the backend team.

Reply:
I verified with the backend team and they said the backend will not prefix positive values with a '+'.